src/components/Pivot/Pivot.tsx

```diff
@@ -0,0 +1,93 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { NestableBaseComponentProps } from '../../util/BaseComponent/props';
+import { Pivot as FabricPivot, PivotItem as FabricPivotItem } from 'office-ui-fabric-react/lib/Pivot';
+
+import './Pivot.css';
+
+export interface PivotItem {
+  ariaLabel?: string;
+  key: string;
+  text: string;
+}
+
+export interface PivotProps extends NestableBaseComponentProps {
+  /**
+   * Indicates whether the text on the tabs will be rendered on white colors
+   * to be used with dark backgrounds.
+   */
+  isDark?: boolean;
+  /**
+   * Indicates the key of the initially selected tab or pivot item.
+   */
+  selectedKey: string;
+  /**
+   * Contains the information of the pivot items that will be rendered within this component.
+   */
+  pivotItems: PivotItem[];
+  /**
+   * Callback that will be executed every time there is a change in the selected pivot item.
+   */
+  onChange(key: string): void;
+}
+
+export interface PivotState {
+  /**
+   * Key of the selected pivot item at a certain point.
+   */
+  selectedKey: string;
+}
+
+export default class Pivot extends React.Component<PivotProps, PivotState> {
+  constructor(props: PivotProps) {
+    super(props);
+    this.onLinkClick = this.onLinkClick.bind(this);
```

Comment:
you can just change the method on the class instead of binding in the constructor, here's an example:
https://github.com/Microsoft/YamUI/blob/master/src/components/PreviewCard/PreviewCard.tsx#L246